apps/gateway-email/package.json

```diff
@@ -4,7 +4,6 @@
   "private": true,
   "type": "module",
   "scripts": {
-    "dev": "docker run -p 9000:8080 -e AWS_ACCESS_KEY_ID -e AWS_SECRET_ACCESS_KEY -e AWS_REGION=eu-north-1 gateway-email:latest",
```

Comment:
necessary to be able to run pnpm dev. Didn't mean to push but don't know how to make it work locally with this script

Reply:
Can you rename this and dispatcher-auth's dev scripts to "docker:dev"?